src/demo/share/jfc/Stylepad/HelloWorld.java

```diff
@@ -187,22 +187,22 @@ static class Run {
             new Run("none", "Hello from Cupertino")
         }),
         new Paragraph("title", new Run[] {
-            new Run("none", "\u53F0\u5317\u554F\u5019\u60A8\u0021")
+            new Run("none", "台北問候您!")
         }),
         new Paragraph("title", new Run[] {
-            new Run("none", "\u0391\u03B8\u03B7\u03BD\u03B1\u03B9\u0020" // Greek
-            + "\u03B1\u03C3\u03C0\u03B1\u03B6\u03BF\u03BD"
-            + "\u03C4\u03B1\u03B9\u0020\u03C5\u03BC\u03B1"
-            + "\u03C2\u0021")
+            new Run("none", "Αθηναι " // Greek
+            + "ασπαζον"
+            + "ται υμα"
+            + "ς!")
```

Comment:
Maybe we can merge the string now:
```suggestion
            new Run("none", "Αθηναι ασπαζονται υμας!") // Greek
```
At least two words seem to be split between the wrapped lines.